### DISCUSSION TITLE
Code block execution within the document

### DISCUSSION THREAD

**Discussion by ViperX7** [nvim-neorg/neorg#65] [General]
I know that the feature to evaluate code blocks and print their output within the document is currently
Not a priority feature so I forked *mdeval.nvim* module and added support for executing norg code blocks and finally it's merged you can get the module here https://github.com/jubnzv/mdeval.nvim

To execute your codeblock just move your cursor to the code block and `:mdeval` and code will be executed

**Comment by vhyrro** [comment]
Yo that's pretty neat! This should at least make it a bit easier until we add first class support for code block evaluation ourselves, good job man.
